### ISSUE TITLE
Apps using webview crashing with chromium: [ERROR:gl_fence_egl.cc(62)] Failed to wait for EGLSync. error:EGL_BAD_MATCH

### ISSUE THREAD

**Issue by HighwayStar** [HighwayStar/android_device_xiaomi_mocha#5]
This caused by some bug in old mipad  GLES drivers. Can be resolved by using custom built webview.apk from https://github.com/manumanfred/android_external_chromium-webview

Or by updating graphics drivers. Work both on kernel and nvidia shield blobs required.

**Comment by Nihhaar** [comment]
https://codereview.chromium.org/892323002/
This seems more appropriate patch. (Yeah, change the gl_vendor). But is our gl vendor is Broadcom?

**Comment by HighwayStar** [comment]
Yes, first I've tried to build custom webview myself, but stuck on downloading chromium sources and toolchain. Failed to download gs://chromium-android-tools/play-services/8487000/11cc73d4b7fa82560fbf5bbc1095dbac30308e7c

when using this buildscript https://gist.github.com/luca020400/aca28ac562d1de54fe94688d11578812

**Comment by HighwayStar** [comment]
Yes, mipad specific patch shows much better performance. Fixes most of performance and incompatibility issues (Gmail, Google Books now works fine).  Patch based on found in @Nihhaar  gists one https://github.com/HighwayStar/android_external_chromium-webview/blob/cm-14.1-mocha/patches/mipad-patch.patch

Built webview apk located here. https://github.com/HighwayStar/android_external_chromium-webview/tree/cm-14.1-mocha/prebuilt/arm

**Comment by Nihhaar** [comment]
Actually, I wrote the patch based on Nexus 9's similar issue which was reported in chromium bugs, but good to see working well for us.
